### PR TITLE
Add key rotation subcommand in TXS

### PR DESCRIPTION
**Pull request by lpgeiger**
- [x] Improves user experience of key rotation (using a different AuthenticationKey to control an account).

### CODE REVIEW

File: ol/onboard/src/commands/mnem_info_cmd.rs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//! `version` subcommand
```
Comment (simsekgokhan) [inline ol/onboard/src/commands/mnem_info_cmd.rs:1]: `keygen` instead of `version` ?

File: ol/onboard/src/commands/mnem_info_cmd.rs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+#![allow(clippy::never_loop)]
+
+use abscissa_core::{Command, Options, Runnable};
+use ol_keys::wallet;
+/// `keygen` subcommand
+#[derive(Command, Debug, Default, Options)]
+pub struct KeygenCmd {}
```
Comment (simsekgokhan) [inline ol/onboard/src/commands/mnem_info_cmd.rs:9]: Shouldn't the filename be "keygen.rs"?

File: ol/onboard/src/commands/mnem_info_cmd.rs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+
+impl Runnable for KeygenCmd {
+    /// Print version message
```
Comment (simsekgokhan) [inline ol/onboard/src/commands/mnem_info_cmd.rs:13]: Forgot to remove?

File: ol/txs/src/commands/authkey_cmd.rs [hunk 4]
```diff
@@ -0,0 +1,43 @@
+//! `version` subcommand
```
Comment (simsekgokhan) [inline ol/txs/src/commands/authkey_cmd.rs:1]: Same problem here, two more places with "version" in this file

File: ol/txs/src/commands/authkey_cmd.rs [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    /// Print version message
+    fn run(&self) {
+        if let Some(key) = &self.new_key {
+            let entry_args = entrypoint::get_args();
+            let tx_params = tx_params_wrapper(TxType::Cheap).unwrap();
+
+            let script = transaction_builder::encode_rotate_authentication_key_script(key.to_vec());
+            maybe_submit(
+                script,
+                // transaction_builder::encode_demo_e2e_script(42),
```
Comment (simsekgokhan) [inline ol/txs/src/commands/authkey_cmd.rs:32]: Is this forgotten also?

File: ol/txs/src/commands/version_cmd copy.rs [hunk 6]
```diff
@@ -0,0 +1,16 @@
+//! `version` subcommand
```
Comment (simsekgokhan) [inline ol/txs/src/commands/version_cmd copy.rs:1]: This file also to be deleted? I guess

File: ol/txs/src/commands/authkey_cmd.rs [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+use ol_types::config::TxType;
+
+#[derive(Command, Debug, Default, Options)]
+pub struct AuthkeyCmd {
+    #[options(help = "the authkey to rotate to")]
+    new_key: Option<AuthenticationKey>,
+}
+
+impl Runnable for AuthkeyCmd {
+    /// Print version message
```
Comment (simsekgokhan) [inline ol/txs/src/commands/authkey_cmd.rs:22]: One more here :)

**Review by soaresa — APPROVED** [review]
Missing tests?

**Comment by lpgeiger** [comment]
> Missing tests?

This is an upstream functionality, tests exist in Move. Doing an e2e test on this is quite involved, diminishing returns.

**Comment by soaresa** [comment]
Thanks!

**Pull request merged**